File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1154.c
#include <stdio.h>

int main(){
    int idade, n=0;
    double soma=0;
    while(1){
        scanf("%d", &idade);
        if(idade<0){
            break;
        }
        soma+=idade;
        n++;
    }
    printf("%.2lf\n", soma/n);

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1154.cs
using System;

class URI
{
    static void Main() {
        
        int idade, n=0;
        double soma=0;
        while(true){
            idade = int.Parse(System.Console.ReadLine().Trim());
            if(idade<0){
                break;
            }
            soma+=idade;
            n++;
        }
        Console.Write("{0:f2}\n", soma/n);

    }
}